pkgs/applications/virtualization/kata-containers/default.nix

```
@@ -0,0 +1,190 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchurl, pkgconfig
+, libseccomp, yq-go, go, qemu }:
+
+let
+  version = "1.11.2";
+
+  # prepare source to have a structure of GOPATH
+  setSourceRoot = ''
+    mkdir -p "go/src/$(dirname "$goPackagePath")"
+    cp -R source "go/src/$goPackagePath"
+    export sourceRoot="go/src/$goPackagePath"
+  '';
+
+  # default make flags, set GOPATH, GOCACHE and install prefix
+  makeFlags = [
+    "GOPATH=$(NIX_BUILD_TOP)/go"
+    "GOCACHE=$(TMPDIR)/go-cache"
+    "PREFIX=$(out)"
+  ];
+
+  # default meta
+  meta = with stdenv.lib; {
+    homepage = "https://katacontainers.io/";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ offline ];
```

Would this be more correct?

```suggestion
    maintainers = with maintainers; [ offlinehacker ];
```

---

Actually no, this is alias I use, see https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L5971 :smile:

---

Actually i was thinking to change my nick name, but now it's everywhere, so yeah

---

Alright thank you for the clarification.